### PR TITLE
Document adding certificatesmto trust store

### PR DESCRIPTION
**Pull request by alex-domino**
Ref: DOM-4782

### CODE REVIEW

File: _docs/configuration/ssl.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Starting with Domino 2.5, the deployer is able to automatically add multiple SSL certificates to the trust store, minimizing the ammount of manual work required.
+
+## Configuration through the manifest
+
+Multiple SSL certificates can be configured through the following settings:
+```yaml
+domino:
+  trusted_certs:
+    my_certificate: /path/to/my/certificate.crt
+    my_other_certificate: /path/to/other/certificate.crt
```
Comment (fernandoacorreia) [inline _docs/configuration/ssl.md]: This needs realistic examples. e.g. `ssl`, `ldap`, `smtp`.

File: _docs/configuration/ssl.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+- The keys is used to name the certificate file. It is used both inside of Java's certificate store and as the file name for the certificate in the `/domino` folder
+- The value should point to the location of the certificate on the deployer box, as is currently the case with`trusted_ca_cert_file`
```
Comment (fernandoacorreia) [inline _docs/configuration/ssl.md]: I think you mean `before version 2.5` instead of `currently`.

File: _docs/configuration/ssl.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## trusted_ca_cert_file deprecation
+
+As this duplicates the functionality provided by `trusted_ca_cert_file`, the latter is no longer supported by the deployer in the 2.5 release of Domino.
+
+To migrate a manifest for 2.5 while maintaining the functionality, we recommend the following:
```
Comment (fernandoacorreia) [inline _docs/configuration/ssl.md]: Mentions to version 2.5 and how it's different from 2.4, and what people need to to do upgrade to 2.5 need to go under the "Upgrading" section, not under the "Configuration" section, which is meant to describe how to configure the current Domino version.

**Comment by alex-domino** [comment]
Updated

**Comment by alex-domino** [comment]
No longer in line with the state of LDAP certificate support, closing

**Pull request closed without merging**